cmd/podmanV2/containers/commit.go

```diff
@@ -0,0 +1,82 @@
+package containers
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	commitDescription = `Create an image from a container's changes. Optionally tag the image created, set the author with the --author flag, set the commit message with the --message flag, and make changes to the instructions with the --change flag.`
+
+	commitCommand = &cobra.Command{
+		Use:     "commit [flags] CONTAINER [IMAGE]",
+		Short:   "Create new image based on the changed container",
+		Long:    commitDescription,
+		RunE:    commit,
+		PreRunE: preRunE,
+		Example: `podman commit -q --message "committing container to image" reverent_golick image-committed
+  podman commit -q --author "firstName lastName" reverent_golick image-committed
+  podman commit -q --pause=false containerID image-committed
+  podman commit containerID`,
+	}
+
+	// ChangeCmds is the list of valid Changes commands to passed to the Commit call
+	ChangeCmds = []string{"CMD", "ENTRYPOINT", "ENV", "EXPOSE", "LABEL", "ONBUILD", "STOPSIGNAL", "USER", "VOLUME", "WORKDIR"}
+)
+
+var (
+	commitOptions = entities.CommitOptions{
+		ImageName: "",
+	}
+	iidFile string
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: commitCommand,
+	})
+	flags := commitCommand.Flags()
+	flags.StringArrayVarP(&commitOptions.Changes, "change", "c", []string{}, fmt.Sprintf("Apply the following possible instructions to the created image (default []): %s", strings.Join(ChangeCmds, " | ")))
```

Comment:
```suggestion
	flags.StringArrayVarP(&commitOptions.Changes, "change", "c", []string{}, "Apply the following possible instructions to the created image (default []): " + strings.Join(ChangeCmds, " | "))
```
Is faster and uses less resources.